README.md

```diff
@@ -212,8 +214,99 @@ e.g: `@Mixpanel\Expression("<expression>")` or `@Mixpanel\Set("<property>", valu
 **Note**: All `id` properties can be omitted, as they will be set with the id of
 the current user in `security.context`
 
+### MixpanelEvent
+
+You can also send an event through symfony events when the annotations are not sufficient like this:
+```php
+#In controller
+namespace myNamespace;
+
+use Gordalina\MixpanelBundle\Annotation as Annotation;
+use Gordalina\MixpanelBundle\Mixpanel\Event\MixpanelEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+// ...
+
+public function edit(User $user, EventDispatcherInterface $eventDispatcher, Request $request)
+{
+    // Your code
+    $annotation = new Annotation\Track();
+    $annotation->event = 'My event';
+    $annotation->props = [
+        'prop 1' => 'data 1',
+        'prop 2' => 'data 2',
+    ];
+    
+    $eventDispatcher->dispatch(new MixpanelEvent($annotation, $request));
+    // Rest of your code
+}
+```
+
+### Override Props in all Annotations
+
```

Comment:
Here's a great place to add the example of the user id being injected automatically.